bindings/nodejs/examples/presign.js

```diff
@@ -18,24 +18,29 @@
  */
 
 const http = require('node:http')
-const url = require('node:url')
 const { Operator } = require('../index')
 
 const op = new Operator('s3', {
   root: '/',
-  bucket: 'example-bucket',
+  bucket: 'presign'
 })
 
 const server = http.createServer(async (req, res) => {
   res.setHeader('Content-Type', 'text/json; charset=utf-8')
-
-  if (req.url.startsWith('/presign') && req.method === 'GET') {
-    const urlParts = url.parse(req.url, true)
-    const path = urlParts.query.path
-    const expires = urlParts.query.expires
-
-    const presignedRequest = op.presignRead(path, parseInt(expires))
-
+  if (req.url.startsWith('/presign/download')) {
+    const url = new URL(req.url, 'http://locahost:3000')
+    const path = url.searchParams.get('path')
+    const expires = parseInt(url.searchParams.get('expires'), 10)
+    const presignedRequest = op.presignRead(path, expires)
+    /* Redirect client to download file from presigned url */
+    res.statusCode = 307
+    res.setHeader('Location', presignedRequest.uri)
```

Comment:
```suggestion
    res.setHeader('Location', presignedRequest.url)
```

Sorry for updating the API. Please check the latest file.